docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| №  | Description                                                                                                                                                                                                                                              | Requirement | Commentary                                                                         |
+| -- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- | ---------------------------------------------------------------------------------- |
+|    | 1.1 Favorite empty opening page 11                                                                                                                                                                                                                       |             |                                                                                    |
+| 1  | Text "here will be your added words as soon as you create tags and start adding favorites" is presented                                                                                                                                                  |             |                                                                                    |
+| 2  | Red button "create a tag" is shown in a middle of the screen                                                                                                                                                                                             |             |                                                                                    |
+| 3  | Pressing a button is leading to opening a favourites_create a tag (12) page                                                                                                                                                                              |             |                                                                                    |
+| 4  | Image is the same as on model                                                                                                                                                                                                                            |             |                                                                                    |
```

Comment:
What do you mean by 'model'? If you mean design, you can call it design or mockups